variants/fcos/okd-rpms/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM release:artifacts as artifacts
+FROM release:machine-config-operator as mcd
+
+FROM fedora:37
+ARG CRIO_VERSION=1.25
+
+WORKDIR /rpms
+
+COPY --from=artifacts /srv/repo/* /tmp/rpms
+
+RUN \
+    mkdir -p /rpms && \
+    mv -v /tmp/rpms/$([ -d /tmp/rpms/$(uname -m) ] && echo $(uname -m)/)*.rpm /rpms/ && \
```

Comment:
is it worth adding the rpms in the `artifacts` and adding an imagestreamtag referencing it with name okd-rpms?

In future, the two names should converge.

Reply:
wouldn't like to break any builds which depend on original `quay.io/coreos-assembler/fcos:next-devel`, if any

Reply:
as i understand correctly scos would be like rhel coreos and it will contain any necessary rpms (including crio), its a reason when fcos variant is a little separated at the momemnt (or opposite)

Reply:
The artifacts image is not part of the payload but an auxiliary image. 

Adding the cri-o rpms and using some symbolic links to refer the rpms where they are currently stored should be enough if I understand correctly the use case. The consumers of the rpms will use the rpms at need... 

https://github.com/okd-project/okd-payload-pipeline/blob/main/forks/artifacts/Dockerfile.centos9

Reply:
there are still some references to `GetOKDRPMSImage` within assisted service codebase:
```
https://github.com/openshift/assisted-service/blob/99125bfa925143394e32f785e104c63624d1f601/internal/oc/release.go#L45
https://github.com/openshift/assisted-service/blob/99125bfa925143394e32f785e104c63624d1f601/internal/ignition/ignition.go#L1627
```
finally the `GetOKDRPMSImage` refers to `okdRPMSImageName  = "okd-rpms"` as tag for registry/okd/okd-contents:4.12.0-0.okd-xxxx-okd-rpms
and this is a point where scos might be more suitable: full control for OS version/build and ostree (including crio, kubelet, cli, ncat, etc) without temporary rpm overlaying on bootkube/bootstrap procedure